sdk/tables/azure-data-tables/azure/data/tables/_table_service_client.py

```diff
@@ -370,9 +356,6 @@ def get_table_client(self, table_name, **kwargs):
             self.url,
             table_name=table_name,
             credential=self.credential,
-            key_resolver_function=self.key_resolver_function,
-            require_encryption=self.require_encryption,
-            key_encryption_key=self.key_encryption_key,
             api_version=self.api_version,
             transport=self._client._client._pipeline._transport,  # pylint: disable=protected-access
             policies=self._policies,
```

Comment:
Discussed this out of band a few days ago, are we using the `_pipeline` we build a few lines before this.

Reply:
I hadn't _planned_ to fix that in this PR - however I ended up needing to for the retry tests - so this should be be resolved :)